--- prewikka-5.2.0/setup.py.orig	2020-09-09 16:44:23.000000000 +0200
+++ prewikka-5.2.0/setup.py	2024-09-19 18:19:38.534660434 +0200
@@ -39,7 +39,7 @@ LIBPRELUDE_REQUIRED_VERSION = "5.2.0"
 LIBPRELUDEDB_REQUIRED_VERSION = "5.2.0"
 
 
-def init_siteconfig(conf_prefix, data_prefix):
+def init_siteconfig(conf_prefix, data_prefix, build_lib):
     """
     Initialize configuration file (prewikka/siteconfig.py).
 
@@ -54,7 +54,7 @@ def init_siteconfig(conf_prefix, data_pr
         ('libpreludedb_required_version', LIBPRELUDEDB_REQUIRED_VERSION),
     )
 
-    with open('prewikka/siteconfig.py', 'w') as config_file:
+    with open(os.path.join(build_lib, 'prewikka/siteconfig.py'), 'w') as config_file:
         for option, value in configuration:
             config_file.write("%s = '%s'\n" % (option, value))
 
@@ -74,7 +74,7 @@ class MyDistribution(Distribution):
 class my_install(install):
     def finalize_options(self):
         # if no prefix is given, configuration should go to /etc or in {prefix}/etc otherwise
-        if self.prefix:
+        if self.prefix and self.prefix != '/usr':
             self.conf_prefix = self.prefix + "/etc/prewikka"
             self.data_prefix = self.prefix + "/var/lib/prewikka"
         else:
@@ -112,7 +112,7 @@ class my_install(install):
         self.install_conf()
         self.install_wsgi()
         self.create_datadir()
-        init_siteconfig(self.conf_prefix, self.data_prefix)
+        init_siteconfig(self.conf_prefix, self.data_prefix, self.build_lib)
         install.run(self)
 
         os.chmod((self.root or "") + self.conf_prefix, 0o755)
@@ -182,7 +182,7 @@ class PrewikkaTest(TestCommand):
         self.test_suite = True
 
     def run_tests(self):
-        init_siteconfig('conf', 'tests/downloads')
+        init_siteconfig('conf', 'tests/downloads', '')
 
         import pytest  # import here, cause outside the eggs aren't loaded
 
