--- prewikka-5.2.0/prewikka/localization.py.orig	2024-09-19 10:35:19.108921231 +0200
+++ prewikka-5.2.0/prewikka/localization.py	2024-09-19 18:18:41.388303356 +0200
@@ -56,7 +56,7 @@ class TranslationProxy(object):
         self._catalogs_lock = Lock()
 
         self._domains_lock = Lock()
-        self._domains = collections.OrderedDict([("prewikka", pkg_resources.resource_filename(__name__, "locale"))])
+        self._domains = collections.OrderedDict([("prewikka", "/usr/share/locale")])
 
     def add_domain(self, domain, locale_dir):
         with self._domains_lock:
